models/cuboid/Cargo.toml

```diff
@@ -3,5 +3,11 @@ name = "cuboid"
 version = "0.1.0"
 edition = "2021"
 
-[dependencies.fj]
-path = "../../crates/fj"
+[[bin]]
+name = "cuboid"
+path = "src/main.rs"
+
+[dependencies]
+fj = { path = "../../crates/fj" }
```

Comment:
Why change the formatting for the existing dependency, in addition to adding the new ones? Personally, I like the previous style better, as it's a bit less dense, which makes it a bit easier to read for me.

But it's a matter of taste. I wouldn't have complained if just the new dependency were formatted that way, but adding new code and making changes to existing in the same commit, makes this harder to review than it needs to be. If you want to make changes to existing code that aren't directly related, I would prefer having that in a separate commit, with an explanation on why the change is made.